Sources/SATSCore/Assets/ThemeColors.xcassets/Blue Theme/Primary/bluePrimaryHighlight.colorset/Contents.json

```diff
@@ -23,9 +23,9 @@
         "color-space" : "srgb",
         "components" : {
           "alpha" : "1.000",
-          "blue" : "0xE1",
-          "green" : "0xA5",
-          "red" : "0x65"
+          "blue" : "0xCC",
+          "green" : "0x8A",
+          "red" : "0x30"
```

Comment:
Just out of curiosity, what are you using this color for? It's called “hover” in Figma, and we haven't added it to the Android design system because we don't do hover effects.

Reply:
Right! Glad to hear you've started adding those states. I get so confused watching casts from iOS devices and then suddenly screens just popping in without any indication that anything was pressed. Good on you. :+1: